mne_qt_browser/_pg_figure.py

```diff
@@ -1800,6 +1802,32 @@ def __init__(self, main, title="Settings", **kwargs):
         self.scroll_sensitivity_slider.setValue(self.mne.scroll_sensitivity)
         layout.addRow("horizontal scroll sensitivity", self.scroll_sensitivity_slider)
 
+        # Add subgroup box to show channel type scalings
+        layout.addItem(QSpacerItem(10, 10, QSizePolicy.Minimum, QSizePolicy.Expanding))
+        ch_scaling_box = QGroupBox("Channel Type Scalings")
+        ch_scaling_box.setStyleSheet("QGroupBox { font-size: 12pt; }")
+        ch_scaling_layout = QFormLayout()
+        self.ch_scaling_spinboxes = {}
+
+        # Get all unique channel types
+        ordered_types = self.mne.ch_types[self.mne.ch_order]
+        unique_type_idxs = np.unique(ordered_types, return_index=True)[1]
+        ch_types_ordered = [ordered_types[idx] for idx in sorted(unique_type_idxs)]
+        for ch in ch_types_ordered:
+            if ch in self.mne.unit_scalings.keys():
+                ch_spinbox = QDoubleSpinBox()
+                ch_spinbox.setRange(-float("inf"), float("inf"))
+                ch_spinbox.setDecimals(3)
+                ch_spinbox.setValue(self.mne.scalings[ch] * self.mne.unit_scalings[ch])
+                ch_spinbox.setDisabled(True)
+                # ch_spinbox.setReadOnly(True)
```

Comment:
Could you remove this comment if the command is unused in one of the follow-up PR?